Dockerfile

```diff
@@ -10,13 +10,19 @@
 #   Red Hat, Inc. - initial API and implementation
 #
 
-FROM golang:1.10.3 as builder
+FROM golang:1.12-alpine3.9 as builder
+RUN apk add --no-cache ca-certificates
+RUN adduser -D -g '' appuser
 WORKDIR /go/src/github.com/eclipse/che-jwtproxy/
-COPY . .
+COPY . /go/src/github.com/eclipse/che-jwtproxy/
 RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-w -s' -a -installsuffix cgo -o jwtproxy cmd/jwtproxy/main.go
 
 
-FROM alpine:3.7
+
+FROM alpine:3.9
+USER appuser
+COPY --from=builder /etc/passwd /etc/passwd
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 ENV XDG_CONFIG_HOME=/che-jwtproxy-config/
 VOLUME /che-jwtproxy-config
```

Comment:
Why do we need this volume? I'm not sure that in the end it will be rewritten with Pod Volume..

Reply:
IMHO, we don't need this volume. We mount it from the k8s side so it doesn't have to be present in the Dockerfile.

Reply:
Unsure if it's a plus or minus, but having a volume in the dockerfile means (some) OpenShift clusters will show an error event if the pod doesn't mount a corresponding volume.